integration/kubernetes/overhead/test_ntttcp/workloads.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: overhead
+  labels:
+    app: overhead
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: overhead
+  template:
+    metadata:
+      labels:
+        app: overhead
+    spec:
+      runtimeClassName: kata-qemu
+      containers:
+      - name: overhead
+        image: jcvenega/ntttcp
```

Comment:
@GabyCT  who owns  the kata dockerhub account probably need to put this in a Dockerfile and let dockerhub built it for us.

Reply:
@jcvenegas no idea maybe @egernst knows